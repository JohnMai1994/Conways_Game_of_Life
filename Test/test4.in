new 10
init
5 5 4 4 3 3 6 6 7 7
2 2 1 1 0 0 8 8 9 9
-1 -1
print

step
print
step
print
step
print
step
print
step
print

