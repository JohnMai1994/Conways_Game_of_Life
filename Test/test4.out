X_________
_X________
__X_______
___X______
____X_____
_____X____
______X___
_______X__
________X_
_________X
__________
_X________
__X_______
___X______
____X_____
_____X____
______X___
_______X__
________X_
__________
__________
__________
__X_______
___X______
____X_____
_____X____
______X___
_______X__
__________
__________
__________
__________
__________
___X______
____X_____
_____X____
______X___
__________
__________
__________
__________
__________
__________
__________
____X_____
_____X____
__________
__________
__________
__________
__________
__________
__________
__________
__________
__________
__________
__________
__________
__________
